src/components/deviceprofileBuilder.ts

```diff
@@ -104,6 +106,15 @@ function getDirectPlayProfiles(): Array<DirectPlayProfile> {
             });
         }
 
+        // mkv profile: surround and normal codecs
```

Comment:
I haven't commented clearly enough that chromecast does not really support this and it may fail on some files.